TeamCode/src/main/java/org/firstinspires/ftc/teamcode/auton/Tests/MotorEncoderTest.java

```diff
@@ -0,0 +1,94 @@
+package org.firstinspires.ftc.teamcode.auton.Tests;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.OpMode;
+import com.qualcomm.robotcore.hardware.DcMotor;
+
+import org.firstinspires.ftc.teamcode.subsystems.hardware.*;
+
+@Autonomous(name="Motor Encoder Test")
+
+public class MotorEncoderTest extends OpMode {
+    float wheelEncoderPPR = 537.7f; // PPR
+    int wheelDiameter = 96; // mm
+    double mmPerEncoderTick = (360/wheelEncoderPPR)/360*(wheelDiameter*Math.PI); // 0.56089435511 mm
+    double distance;
+    float motorRunPower = 0.4f;
+    private Hardware robot;
+
+    @Override
+    public void init(){
+      robot = new Hardware(hardwareMap, telemetry);
+      robot.introduce(new HardwareElement<>(DcMotor.class, hardwareMap, "leftFront"));
+      robot.introduce(new HardwareElement<>(DcMotor.class, hardwareMap, "leftRear", "setDirection:REVERSE"));
+      robot.introduce(new HardwareElement<>(DcMotor.class, hardwareMap, "rightFront"));
+      robot.introduce(new HardwareElement<>(DcMotor.class, hardwareMap, "rightRear"));
+    }
+
+    @Override
+    public void start() {
+        distance = inchesToMM(12);
+
+        setTargetPosition(robot.<DcMotor>get("leftFront"), distance);
+        setTargetPosition(robot.<DcMotor>get("rightFront"), distance);
+        setTargetPosition(robot.<DcMotor>get("leftRear"), distance);
+        setTargetPosition(robot.<DcMotor>get("rightRear"), distance);
+
+        while (wheelsInPosition()) {
+            updateMotors();
+        }
```

Comment:
Can you abstract this setting target positions and running motors into a method that you call in start? It won't change any of the functionality but it will help clean things up and make it more clear what's going on :)